loadtest/user/userentity/actions.go

```diff
@@ -1515,3 +1591,40 @@ func (ue *UserEntity) prepareRequest(method, url string, data io.Reader, headers
 
 	return rq, nil
 }
+
+func (ue *UserEntity) getGqlResponse(input any) (*graphql.Response, error) {
+	var gqlResp *graphql.Response
+	buf, err := json.Marshal(input)
+	if err != nil {
+		return nil, err
+	}
+
+	req, err := ue.prepareRequest(http.MethodPost,
+		getGQLURL(ue.client.URL),
+		bytes.NewReader(buf),
+		map[string]string{})
```

Comment:
Nit: probably the compiler optimizes this, but what if we declare a single empty map outside of the function and use it here?

Reply:
Hurts readability. Probably zero impact on performance. You are thinking too low-level. :P

Reply:
Fair enough :nerd_face: